CHANGELOG.md

```diff
@@ -9,6 +9,32 @@
   Microcosm classes.
 - Fixed a bug where the first action dispatched would always fire a
   change event, even if state didn't change.
+- Action status changing methods are auto-bound, and will warn when a
+  completed action attempts to move into a new state (strict mode only)
+
+### Autobound action status methods
+
+Action status methods, like `action.resolve()`, or `action.reject()`
+are now auto-bound. This means that you can pass them directly into a
+callback, and the scope of the method will remain as the action. This
```

Comment:
I think rewriting this to `and the value of `this` will remain the action` would clear this up quite a bit.